Engine/Engine/Core/Application.cpp

```diff
@@ -44,11 +44,16 @@ void Application::Init()
 
 	assetMan->SetAssetDir("Assets");
 	assetMan->LoadAssetsFromAssetDir();
-	renderer->SetAssetManager(assetMan);
-	renderer->CreateMeshes();
-	//masterBG.InitSound();
-	//masterBG.LoadFile("audio/gameMusic.mp3");
-	//masterBG.Play(false);
+
+	//TODO: Change this to dynamically create programs
+	renderer->CreateBasicProgram();
```

Comment:
This function doesn't exist anymore in the renderer.... it should be
`renderer->SetAssetManager(assetMan);
renderer->CreateMeshes();`

but GitHub is saying it's changed?